pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -162,13 +162,22 @@ func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
 	return err
 }
 
-func (h *NetworkUtilsHandler) IsIpv6Enabled() bool {
-	podIp := os.Getenv("MY_POD_IP")
-	if !netutils.IsIPv6String(podIp) {
-		log.Log.V(5).Info("Since the pod ip is non IPv6, IPv6 is disabled")
+func (h *NetworkUtilsHandler) IsIpv6Enabled(interfaceName string) bool {
+	link, err := Handler.LinkByName(interfaceName)
+	addrList, err := Handler.AddrList(link, netlink.FAMILY_ALL)
```

Comment:
its a decision that happen in the creation time, what if someone will add ipv6 on the fly,
he will need to create also the routes and whatever this affect, since the decision already took place, isnt it ?
and if so, is it acceptable? or worth to config anyway and thats it ? (in case its possible, which so far seems)

Reply:
Once the net_admin capability will be removed from the virt-launcher there will be no possibly to update the network configuration of the virt-launcher pod by the user. Meanwhile, yes, the user can break the configuration. If the user will change the net config of the pod it can affect the bridge binding that uses this config, it can affect the masquesrage binding (if the k6t-bridge is removed/changed). So I think this change doesn't add extra danger to the existing code. When the net admin cap will be removed it will be even safer than using MY_POD_IP env_var that can be changed any time.